tests/integration/autonaming/step2/index.ts

```diff
@@ -1,13 +1,30 @@
-// Copyright 2016-2018, Pulumi Corporation.  All rights reserved.
+// Copyright 2016-2019, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
 
 import * as k8s from "@pulumi/kubernetes";
 
+export const namespace = new k8s.core.v1.Namespace("test-namespace");
```

Comment:
Should we append a random string to the ns just to be safe? https://stackoverflow.com/a/8084248/2274669

Reply:
The k8s resource will already have a random suffix. This is just the pulumi resource name, and I left `.metadata.name` unspecified, so it will be autogenerated.